src/sdk/accountspace_fs.js

```diff
@@ -25,6 +26,26 @@ const { account_cache } = require('./object_sdk');
 const dummy_region = 'us-west-2';
 const dummy_service_name = 's3';
 
+const account_id_cache = new LRUCache({
+    name: 'AccountIDCache',
+    // TODO: Decide on a time that we want to invalidate
+    expiry_ms: Number(config.ACCOUNTS_ID_CACHE_EXPIRY),
+    /**
+     * @param {{ _id: string, config_fs: import('./config_fs').ConfigFS }} params
+     */
+    make_key: ({ _id }) => _id,
+    load: async ({ _id, config_fs }) => config_fs.get_identity_by_id(_id, CONFIG_TYPES.ACCOUNT,
```

Comment:
Why did you prefer not to use the flag `silent_if_missing`?
I see it was used in many cases (and therefore it changes the code from checking it it is undefined to try-catch clause).

Reply:
because it'll create undefined values inside the cache